__tests__/app.test.js

```diff
@@ -0,0 +1,27 @@
+const request = require('supertest')
+const app = require('../app.js')
+const seed = require("../db/seeds/seed.js")
+const data = require("../db/data/test-data/index.js")
+const db = require('../db/connection.js')
+
+beforeEach(() => seed( data ));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+    test('200: respond with array of topic objects', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response) => {
+            console.log(response.body)
+            expect(response.body.topics.length).toBe(3);
+           response.body.topics.forEach((topics) => {
+                expect(typeof topics.slug).toBe('string');
+                expect(typeof topics.description).toBe('string');
+            });
+        }) 
```

Comment:
A nice happy path test, should add a test for a bad path like `/api/toopics` where you would expect a 404, as you've added this functionality to your code